pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools>=42"]
 build-backend = "setuptools.build_meta"
 
 [tool.black]
-line-length = 99
+line-length = 79
```

Comment:
this was an intentional deviation from PEP8. I think we can keep the original 100 char line length - modern displays rarely have trouble with that.